keps/sig-api-machinery/3903-unknown-version-interoperability-proxy/README.md

```diff
@@ -254,14 +254,24 @@ the number if needed, but upgrades aren't that frequent and few resources are
 changed on releases, so these requests should not be common. We will count them
 with a metric.
 
+There could be a large volume of requests for a specific resource which might result in the identified apiserver to be unable to serve the proxied requests. Although this scenario should not occur too frequently, since resource types that have large request traffic should not undergo upgrades too often.
+
 TODO: security / cert stuff.
 
 ## Design Details
 
-TODO: specific API change (x2)
+Specific API change(s)
+
+1. In [APIServiceSpec](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1/types.go#L50)
+   1. Add a list of serviceableby (alternate API server ids ?) to be referenced to identify which apiserver can handle the incoming request.
```

Comment:
```suggestion
   1. Add a list of serviceableBy (alternate API server ids ?) to be referenced to identify which apiserver can handle the incoming request.
```

I think I call this "alternates" in the text above, can you change that wording too?